woocommerce/woocommerce-ios
# Make FulfillButtonTableViewCell Reusable

Ref #2768. 

I'm planning to reuse this cell for #2768 so I made it reusable.

<img src="https://user-images.githubusercontent.com/198826/95380485-0e269380-08a4-11eb-93ae-5ce71a47c836.png" width="240">

## Testing

Please test that fulfillment still works.

1. Navigate to an order that is in “Processing”. 
2. Tap on “Begin Fulfillment”. 
3. Confirm that the fulfillment screen is shown. 

## Submitter Checklist
